org.eclipse.xtext.xbase.tests/.classpath

```diff
@@ -20,7 +20,7 @@
 			<attribute name="test" value="true"/>
 		</attributes>
 	</classpathentry>
-	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-11/"/>
```

Comment:
am not sure if i am ok with that just for the test

Reply:
This PR aims to the time, when Java-17 is already the required minimum. I assumed this will the case as soon Java-21 is supported? So this is actually a draft.
If you are interested in a PR that bumps the required Java-version for all bundles, I can provide one just for that.